stdlib/LinearAlgebra/src/eigen.jl

```diff
@@ -47,28 +44,45 @@ julia> vals == F.values && vecs == F.vectors
 true
 ```
 """
-struct Eigen{T,V,S<:AbstractMatrix,U<:AbstractVector} <: Factorization{T}
+struct Eigen{T,V,S<:AbstractMatrix,U<:AbstractVector,R<:AbstractVector} <: Factorization{T}
     values::U
     vectors::S
-    Eigen{T,V,S,U}(values::AbstractVector{V}, vectors::AbstractMatrix{T}) where {T,V,S,U} =
-        new(values, vectors)
+    vectorsl::S
+    eerrbd::R
```

Comment:
These names are very cryptic. We're not constrained by Fortran or the need for short identifiers, might as well give them proper names

Reply:
The names are taken from the `LAPACK` docu. What do you think about `valbounds` and `vecbounds`.